nengo_ocl/simulator.py

```diff
@@ -431,6 +431,15 @@ def __init__(self, network, dt=0.001, seed=None, model=None, context=None,
         self._reset_cl_rngs()
         self._probe_step_time()
 
+    @property
+    def model(self):
```

Comment:
It is in Nengo master, but when using OCL with older versions of Nengo it doesn't exist, so this is here just for compatibility.